modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -53,16 +53,20 @@ def deploy(cls, func, num_returns, kwargs):
         func : callable
             A function to call.
         num_returns : int
-            Amount of return values expected from `func`.
-        kwargs : dict
-            Keyword arguments to pass to remote instance of `func`.
+            Amount of return values expected from ``func``.
+        *args : list
+            Additional positional arguments to be passed in ``func``.
+        **kwargs : dict
+            Additional keyword arguments to be passed in ``func``.
 
         Returns
         -------
         ray.ObjectRef or list
             Ray identifier of the result being put to Plasma store.
         """
-        return deploy_ray_func.options(num_returns=num_returns).remote(func, kwargs)
+        return deploy_ray_func.options(num_returns=num_returns).remote(
+            func, *args, kwargs
```

Comment:
It looks strange, for this to be so `**kwargs`, `deploy_ray_func` signature should be changed.

Reply:
Historically we have not been able to pass `**kwargs` and `*args` to a Ray task for implementation reasons. To get around this we pass `kwargs` as a dictionary.

Reply:
Do I understand correctly that such remote call did not work before?

```python
>>> def t2(*args, **kwargs):
...   print(args)  
...   print(kwargs) 
...                
>>> t2_rem = ray.remote(t2)
>>> t2_rem.remote("test_args", 2, 6, test_keyword="some value")
ObjectRef(4ee449587774c1f0ffffffffffffffffffffffff0100000001000000)
>>> (pid=28664) ('test_args', 2, 6)
(pid=28664) {'test_keyword': 'some value'}

```

Reply:
@devin-petersohn, ping